src/test/java/io/jenkins/plugins/analysis/core/quality/QualityGateTest.java

```diff
@@ -282,6 +285,261 @@ void shouldBeFailedWhenTotalFailedAndNewUnstableThresholdIsLowerThanWarningCount
         testThreshold(builder, 2, 2, 3, Result.FAILURE);
     }
 
+    @Test
+    void shouldReturnAnEmptyEvaluation() {
+        QualityGateResult qualityGateResult = mock(QualityGateResult.class);
+        AnalysisResult analysisResult = mock(AnalysisResult.class);
+
+        assertThat(qualityGateResult.getEvaluations(analysisResult, this.createGateWithBuilder())).isEmpty();
+    }
+
+    @Test
+    void shouldReturnEvaluationForIsTotalReached() {
+        ThresholdResult totalThresholdResult = mock(ThresholdResult.class);
+        ThresholdResult totalUnstableThresholdResult = mock(ThresholdResult.class);
+        ThresholdResult newFailedThresholdResult = mock(ThresholdResult.class);
+        ThresholdResult newUnstableThresholdResult = mock(ThresholdResult.class);
+
+        when(totalThresholdResult.isTotalReached()).thenReturn(true);
+        when(totalUnstableThresholdResult.isTotalReached()).thenReturn(true);
+        when(newFailedThresholdResult.isTotalReached()).thenReturn(true);
+        when(newUnstableThresholdResult.isTotalReached()).thenReturn(true);
+
+        AnalysisResult analysisResult = mock(AnalysisResult.class);
+        when(analysisResult.getTotalSize()).thenReturn(10);
+
+        QualityGateResult qualityGateResult = new QualityGateResult(totalThresholdResult, totalUnstableThresholdResult,
+                newFailedThresholdResult, newUnstableThresholdResult);
+
+        QualityGate qualityGate = this.createGateWithBuilder();
+
+        ArrayList<String> evaluations = (ArrayList<String>) qualityGateResult.getEvaluations(analysisResult,
```

Comment:
Dann lieber für die Methode getEvaluations eine List als Return Type

Reply:
changed return type of getEvaluations to List

Reply:
Am besten erst kommentieren, wenn der Fix auch im PR committed ist, sonst ist das für mich verwirrend, da ich jeweils eine Mail bekomme. (GitHub markiert die Kommentare übrigens automatisch als aktualisiert, sobald der Commit diese Stelle ändert, d.h. Sie müssen nicht extra erwähnen, dass Sie etwas angepasst haben)

Reply:
Stimmt das habe ich vergessen! Ich habe die Kommentare von mir eher wie eine Art abhacken benutzt. Tut mir leid für die Verwirrung.